package.json

```diff
@@ -92,7 +92,7 @@
     "react-addons-shallow-compare": "15.3.0",
     "react-addons-update": "15.3.0",
     "react-click-outside": "2.1.0",
-    "react-day-picker": "1.3.2",
+    "react-day-picker": "2.4.1",
```

Comment:
In short you should run the command in the CLI immediately after you update some dependency:

```cli
> make shrinkwarp
```

Thus the `npm-shrinkwrap.json` will be update. Finally just add and commit the changes of this file. But yes, take a look to doc is a good idea.